docs/sdk/pnp/web/modal/initialize.mdx

```diff
@@ -57,6 +57,45 @@ This Web3Auth constructor takes an object with `Web3AuthOptions` as input.
 
 <Web3AuthOptions />
 
+### Adding a Private key provider
+
+#### `privateKeyProvider`
+
+`privateKeyProvider` parameter helps you to connect with various wallet sdks. These are preconfigured RPC clients
+for different blockchains used to interact respective blockchain network.
+
+::: note
+It's mandatory to provide `privateKeyProvider` for your corresponding chain namespace. To know more in-depth about providers,
+have a look at your [Providers reference](/sdk/helper-sdks/providers).
+:::
+
+```javascript
+const chainConfig = {
+  chainId: "0x1", // Please use 0x1 for Mainnet
+  rpcTarget: "https://rpc.ankr.com/eth",
+  displayName: "Ethereum Mainnet",
+  blockExplorerUrl: "https://etherscan.io/",
+  ticker: "ETH",
+  tickerName: "Ethereum",
+  logo: "https://images.toruswallet.io/eth.svg",
+};
+
+const ethereumPrivateKeyProvider = EthereumPrivateKeyProvider({
+  config: {chainConfig}
+});
+
+const web3auth = new Web3Auth({
+  clientId: "", // Get your Client ID from the Web3Auth Dashboard
+  web3AuthNetwork: "sapphire_mainnet",
```

Comment:
you can change text "sapphire_mainnet" to type web3AuthNetwork: 

import { WEB3AUTH_NETWORK } from "@web3auth/base";

WEB3AUTH_NETWORK.SAPPHIRE_MAINNET

Reply:
Done